### ISSUE TITLE
Is it possible to use shell-gpt with huggingface API?

### ISSUE THREAD

**Issue by bitcoinmeetups** [TheR1D/shell_gpt#629]
Is it possible to use shell-gpt with huggingface API? If so please advise.

**Comment by devilcoder01** [comment]
@bitcoinmeetups I don't think so, as Hugging Face may not deliver the same level of performance that OpenAI or LLaMA can provide.
